.env.example

```diff
@@ -0,0 +1,15 @@
+# Flag to switch to the production environment (redirect urls, gateway url, etc)
+NEXT_PUBLIC_IS_PRODUCTION=false
+
+# Latest supported safe version, used for upgrade prompts
+NEXT_PUBLIC_SAFE_VERSION=1.3.0
+
+# Access keys
+NEXT_PUBLIC_INFURA_TOKEN=
+NEXT_PUBLIC_SENTRY_DSN=
+NEXT_PUBLIC_BEAMER_ID=
+
+# Wallet specific variables
+NEXT_PUBLIC_WC_BRIDGE=
+NEXT_PUBLIC_FORTMATIC_KEY=
+NEXT_PUBLIC_PORTIS_KEY=
```

Comment:
Can you please add a trailing new line here? I recommend setting up your IDE to do that automatically.

Reply:
For some reason, the automatical inserting only works for code files 🤷